.github/workflows/ci.yml

```diff
@@ -243,12 +243,72 @@ jobs:
         with:
           path: src/github.com/Microsoft/hcsshim
 
+      - name: Install crictl
+        shell: powershell
+        run: |
+          $VerbosePreference = 'Continue'
+          $ErrorActionPreference = 'Stop'
+
+          $crictlVersion = 'v1.24.2'
+          $url = "https://github.com/kubernetes-sigs/cri-tools/releases/download/${crictlVersion}/crictl-${crictlVersion}-windows-amd64.tar.gz"
+          curl.exe -L --no-progress-meter -o c:\crictl.tar.gz $url
+          tar.exe xf c:\crictl.tar.gz -C "${{ github.workspace }}/bin"
+
+          if ( $LASTEXITCODE ) {
+            Write-Output '::error::Could not install crictl.'
+            exit $LASTEXITCODE
+          }
+
+      - name: Install mingw
+        shell: powershell
+        run: |
+          $VerbosePreference = 'Continue'
+          # dont set $ErrorActionPreference since we want to allow choco install to fail later on
+
+          Write-Output '::group::Update chocolaty'
+          choco upgrade -y chocolatey
+          Write-Output '::endgroup::'
+
+          if ( $LASTEXITCODE ) {
+            Write-Output '::error::Could not update chocolatey.'
+            exit $LASTEXITCODE
+          }
+
+          Write-Output 'Install mingw'
+          # Install sometimes fails when downloading mingw zip from source-forge with:
+          #  "ERROR: The remote file either doesn't exist, is unauthorized, or is forbidden for url"
+          # Issue is with accessing from source-forge, which version 10.3+ do not use, but cannot upgrade versions.
```

Comment:
Not sure I understand this sentence. Are you saying that we can't update to 10.3+?

Reply:
yeah, the integration tests fail when running the race detector
https://github.com/microsoft/hcsshim/actions/runs/4050036368/jobs/6967108412
it may be fixed in newer go versions, but for now i want to hold the version to avoid mucking around with the tests